mitosheet/mitosheet/api/get_code_snippets.py

```diff
@@ -63,4 +73,21 @@
 ]
 
 def get_code_snippets(params: Dict[str, Any], steps_manager: StepsManagerType) -> str:
-    return json.dumps(DEFAULT_CODE_SNIPPETS)
+        # If code snippet environment variables are set, then use them to get the code snippets. 
+        mito_config_code_snippets_version = os.environ.get(MITO_CONFIG_KEY_CODE_SNIPPETS_VERSION)
```

Comment:
Can we write a test for this? In mitosheet/tests/api -- I think we can start writing tests for all these (I already wrote one for code snippets).
1. one that additional tests if the URL is provided that is legit
2. one where the Url gives code snippets that aren't the exact format

Both can be done with the JSON echo service you used in the spec, methinks!

Reply:
I can't get the JSON echo service to return a list of json, so I can't use it to test that it works properly, but I can use it to test that it errors when it should